### PR TITLE
Completed Smash Magazin Layout

### PR DESCRIPTION
**Pull request by bmuhamood**
We have completed The Project requirements. 

1. The web structure has been set correctly
2. Used HTML, CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+            <ul>
+                <li><a href="#"><img class="logo" src="./img/logo.png" alt="logo"></a></li>
+                <li> <a href="#">Articles</a></li>
+                <li><a href="#">Guides</a></li>
+                <li> <a href="#">Books</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Membership</a></li>
+                <li><a href="#">More &nbsp;&nbsp;</a></li>
+                <!-- <li><button type="button">More <img class="more-icon" src="./img/more-icon.png" alt="more-icon">
+                    </button></li> -->
```
Comment (tirthajyoti-ghosh) [inline index.html]: Please remove the commented code. Use got history to access previous code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+            </div>
+            <div class="article-1">
+                <div class="article-title">
+                    <div class="author-pic"></div>
+                    <div class="author-name"></div>
+                </div>
+                <div class="article-header"></div>
+                <div class="article-comment"></div>
+                <div class="hashtags"></div>
+            </div>
```
Comment (tirthajyoti-ghosh) [inline index.html]: Use `article` tags instead of `div.article-n` for all the articles listed on the page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    </header>
+    <main>
+        <div class="title">
+            <div class="main-title">
+                <p>Don’t Miss These Latest Articles</p>
+            </div>
+            <div class="title-btns">
+                <button type="button" class="round-btn"></button>
+                <button type="button" class="seriously-red-btn">Seriously, red?</button>
+            </div>
```
Comment (tirthajyoti-ghosh) [inline index.html:39]: This section is not centered 👇

| Source Page	| Clone Page |
|:---:|:---:|
|image|Annotation 2020-07-29 001749|

File: index.html [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+        </div>
+        <div class="membership-2">
+            <div class="membership-head-2">
+                Meet “Click!”: Encourage Clicks
+                Without Shady Tricks
+            </div>
+            <div class="membership-body"></div>
+            <div class="membership-btn"></div>
+            <div class="membership-footer"></div>
+        </div>
```
Comment (tirthajyoti-ghosh) [inline index.html:143]: This section is overflowing.

Annotation 2020-07-29 002322

Annotation 2020-07-29 002435

File: index.html [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+
+        <div class="latest-articles">
+            SEE ALL GUIDES
+        </div>
+
+        <footer>
+            <div class="footer-head"></div>
+            <div class="footer-body"></div>
+            <div class="footer-foot"></div>
+        </footer>
```
Comment (tirthajyoti-ghosh) [inline index.html:251]: Increase the height of the footer.

| Source Page	| Clone Page |
|:---:|:---:|
|image|image|

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## What a remarkable project you have built 💯! This is impressive 🤩👌🧨!

### But in order to get it approved, we need to work on some changes. Let's see...

- Your PR can be a bit more descriptive, eg., what you have done in this project (even if you and I know what this project is, it will be good practice for you down the road).

_I have left some comments down below. Please check them out._

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
#### [REQUIRED]

- Use `article` tags instead of `div.article-n` for all the articles listed on the page. I have pointed out at one place above. 
- Please remove the red color from your heatmap. You can only use grayscale colors.
- Remember to avoid committing commented codes.
- **IMPORTANT!** Remember to NOT use `overflow-x: hidden` in order to remove the horizontal scrollbar. 

Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻 

<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

**Comment by bmuhamood** [comment]
- Changes made based on the review received and done accordingly. 
- In this this projected, we used mostly Grid and Flexbox to put elements in the correct positions. 
- Removed all the commented codes

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Approved ✅

## You have done an incredible job making this project 👏🧨💪! Just perfect 💯! 

### You have met all the requirements of the project. Go ahead and merge it! Onto the next project 🚀 !

Happy coding 👨‍💻 

<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

**Comment by bmuhamood** [comment]
Really Appreciated a lot for kind review and guiding us.

**Pull request merged**